cadence/contracts/bridge/FlowEVMBridgeConfig.cdc

```diff
@@ -181,6 +212,38 @@ contract FlowEVMBridgeConfig {
         return &self.typeToTokenHandlers[type]
     }
 
+    /*****************
+        Constructs
+     *****************/
+
+    /// Entry in the registeredTypes mapping, associating a Type with an EVMAddress and its operational status. Since
+    /// the registeredTypes mapping is indexed on Type, this struct does not additionally store the Type to reduce
+    /// redundant storage.
+    ///
+    access(all) struct TypeEVMAssociation {
+        /// The EVMAddress associated with the Type
+        access(all) let evmAddress: EVM.EVMAddress
+        /// Flag indicating whether operations for the associated Type are paused
+        access(all) var isPaused: Bool
+
+        init(associated evmAddress: EVM.EVMAddress) {
```

Comment:
What does `associated evmAddress` mean? looks like incorrect syntax

Reply:
Cadence has support for [argument labeling](https://cadence-lang.org/docs/1.0/solidity-to-cadence#argument-labelling), similar to [Swift](https://docs.swift.org/swift-book/documentation/the-swift-programming-language/functions#Specifying-Argument-Labels) though it's not well documented in the lang ref. This same question was raised by the auditors during their review, so maybe I should just remove this pattern.